project/BenchmarkWorkFlow.scala

```diff
@@ -26,6 +28,7 @@ object BenchmarkWorkFlow {
           Map(
             "repository" -> "dream11/FrameworkBenchmarks",
             "path"       -> "FrameworkBenchMarks",
+            "ref"        -> "fork-poc",
```

Comment:
Remove this line after

- [ ] Approach Review
- [ ] Merge `fork-poc` to `master` on framework repo